OPHD/UI/WarehouseInspector.cpp

```diff
@@ -84,25 +78,19 @@ void WarehouseInspector::update()
 
 	Window::update();
 
-	auto& renderer = Utility<Renderer>::get();
-
 	ProductPool& pool = mWarehouse->products();
 
-	const auto drawTitleText = [&renderer](NAS2D::Point<int> position, const std::string& title, const std::string& text, int offset = 100) {
-		renderer.drawText(*FONT_BOLD, title, position, NAS2D::Color::White);
-		position.x() += offset ? offset : (FONT_BOLD->width(title) + 20);
-		renderer.drawText(*FONT, text, position, NAS2D::Color::White);
-	};
+	const int labelWidth = 100;
 
 	auto position = mRect.startPoint() + NAS2D::Vector{constants::MARGIN, 25};
-	drawTitleText(position, "Storage", std::to_string(pool.availableStorage()) + " / " + std::to_string(pool.capacity()), 0);
+	drawLabelAndValueLeftJustify(position, labelWidth, "Storage:", std::to_string(pool.availableStorage()) + " / " + std::to_string(pool.capacity()));
```

Comment:
Small note that this changes the positioning of the value and adds a ":" to the title. Probably a good change. Just noting it for visual inspection.

Reply:
That was on purpose on my part. Could have been a separate commit for clarity.